python/cusignal/waveforms/waveforms.py

```diff
@@ -400,7 +406,9 @@ def gausspulse(
 )
 
 
-def chirp(t, f0, t1, f1, method="linear", phi=0, vertex_zero=True):
+def chirp(
+    t, f0, t1, f1, method="linear", phi=0, vertex_zero=True, type=cp.float64
```

Comment:
Let's change the `type` input parameter to accept a string ('real' vs `complex`) rather than an actual cupy dtype. It's more in line with what other libraries (scipy, matlab) do as well. Otherwise, looks good!

Reply:
I'm confused. I thought I had that... let me double check when I get home